lib/components/narrative/trip-tools.js

```diff
@@ -55,11 +56,13 @@ class CopyUrlButton extends Component {
   render() {
     return (
       <div>
-        <Button
-          aria-live={this.state.showCopied ? 'assertive' : undefined}
-          className="tool-button"
-          onClick={this._onClick}
-        >
+        {/* Announces copy button status to AT */}
+        <InvisibleA11yLabel aria-live="assertive">
+          {this.state.showCopied && (
+            <FormattedMessage id="components.TripTools.linkCopied" />
+          )}
+        </InvisibleA11yLabel>
+        <Button className="tool-button" onClick={this._onClick}>
           {this.state.showCopied ? (
             <span>
               <IconWithText Icon={Check}>
```

Comment:
isn't this the same button again?

Reply:
Yes the button itself hasn't changed just removing the `aria-live` element from the button itself and adding to the `div`!

Reply:
Yes but if you look at the file in the source code, it's there twice in a row!

Reply:
Never mind that's just us handling state poorly